### ISSUE TITLE
Avoid innerHTML without checking the element

### ISSUE THREAD

**Issue by adlk** [avatarkava/recipe-todoist#1]
When you are using innerHTML on an object that doesn't exist, you will receive a TypeError

Instead of 
```js
let todayCount = document.querySelector('#top_filters .counter.overdue_color').innerHTML;   
let inboxCount = document.querySelector('#filter_inbox .counter').innerHTML; 
```

do

```js
let todayCount = 0;
let inboxCount = 0;

const todayCountElement = document.querySelector('#top_filters .counter.overdue_color');
const inboxCountElement = document.querySelector('#filter_inbox .counter');

if (todayCountElement) {
  todayCount = parseInt(todayCountElement.innerHTML, 10);
}

if (inboxCountElement) {
  todayCount = parseInt(inboxCountElement.innerHTML, 10);
}
```
This is untested but you might get the idea.

Just imagine what would happen if 1000 users are generating 2 TypeErrors in the Franz loop (that runs every second) – especially services that use tools like sentry.io might start to complain (been there done that 👼)

---
Also, this is just a styleguide thing, but use `let` when you need to override the var. If not, use `const`.

**Comment by avatarkava** [comment]
Should be fixed by 50ddbfb325932b718496b28e83ed3005bebda68a

This does have some opinionated choices about what counters to include on the badge, so I'm open to input on those.

**Comment by avatarkava** [comment]
Insane typo, but should be fixed in latest commit.  I need sleep 😴
